nilearn/reporting/data/html/report_body_template_niftispheresmasker.html

```diff
@@ -11,29 +11,37 @@
         {{js_content}}
       </script>
 
-      <div align="center" class="btn-container">
-        <button id="prev-btn-{{unique_id}}" class="btn">Previous Sphere</button>
-      </div>
-
-      <div align="center" class="btn-container">
+      <div
+        class="pure-g d-flex justify-content-center align-items-stretch mt-2"
+      >
+        <button
+          class="btn pure-u-1-3 text-center rounded-2"
+          id="prev-btn-{{unique_id}}"
+          style="height: 70px"
+        >
+          Previous Sphere
+        </button>
         <h3
```

Comment:
unelegant word wrapping:

![image](https://github.com/user-attachments/assets/3afbfbfe-371d-40a8-b46a-2df1082add56)


will drop to using h4